packages/beacon-node/test/spec/specTestVersioning.ts

```diff
@@ -15,7 +15,7 @@ import {DownloadTestsOptions} from "@lodestar/spec-test-util";
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
 export const ethereumConsensusSpecsTests: DownloadTestsOptions = {
-  specVersion: "v1.4.0-beta.0",
+  specVersion: "v1.4.0-beta.1",
```

Comment:
Out of curiosity, why do we have this magic string in this file and in `packages/params/test/e2e/ensure-config-is-synced.test.ts`?

Reply:
this corresponds to protocol spec test releases (which go hand in hand with spec releases) : https://github.com/ethereum/consensus-spec-tests/releases